src/GameLoveLetter/CardDeck.cs

```diff
@@ -0,0 +1,90 @@
+﻿using GameLoveLetter.Cards;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace GameLoveLetter
+{
+	public class CardDeck
+	{
+		public List<ICard> Cards { get; private set; }
```

Comment:
Maybe use an inheritance from `List` then add your methods to the list, I think it's more simple to use for manipulate the deck

Reply:
Hum, you're right.